app/assets/javascripts/discourse/views/topic-list-item.js.es6

```diff
@@ -10,8 +10,17 @@ export default Discourse.View.extend(StringBuffer, {
        ],
 
   unboundClassNames: function(){
-    var classes = [];
     var topic = this.get('topic');
+    var classes = [];
+    
+    if(topic.get('category')){
+      var category, category_name, category_class;
+      category = topic.get('category');
+      category_name = category.get('name').toLowerCase();
+      category_class = category_name + '-category';
```

Comment:
what if a category name has a space in it or something that needs url
encoding? I am uncertain about this change, feels like custom theme
territory.

On Mon, Jan 26, 2015 at 10:06 PM, Régis Hanol <notifications@github.com>
wrote:

> In app/assets/javascripts/discourse/views/topic-list-item.js.es6
> <https://github.com/discourse/discourse/pull/3129#discussion_r23523253>:
>
> >      var topic = this.get('topic');
> > +    var classes = [];
> > +
> > +    if(topic.get('category')){
> > +      var category, category_name, category_class;
> > +      category = topic.get('category');
> > +      category_name = category.get('name').toLowerCase();
> > +      category_class = category_name + '-category';
>
> This should be replaced by this oneliner
>
> var className = "category-" + topic.get('category.nameLower');
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/discourse/discourse/pull/3129/files#r23523253>.
>